arches/app/views/search.py

```diff
@@ -211,6 +216,7 @@ def get_resource_model_label(result):
         return ""
 
 
+@method_decorator(group_required("Resource Exporter"), name="dispatch")
```

Comment:
I see the issue with the decorator. It should actually just be `@group_required("Resource Exporter")` because it is directly on the view method rather than a view class.